cognite/dm_clients/domain_modeling/schema.py

```diff
@@ -17,6 +17,10 @@
 
 
 logger = logging.getLogger(__name__)
+AUTO_GENERATED_COMMENT = """# THIS FILE IS AUTO-GENERATED!
+# Use `dm_clients schema render` to update it, see `dm_clients --help` for more information.
+
+"""
```

Comment:
This comment is now duplicated when generating the schema using the cli tool `dm_clients schema render`. We should redo the scripts in pure Python, but for now maybe we can just remove the comment from `dm_clients/bin/schema_render.sh`?

Reply:
done.